openshift/scripts/oc_provision_backup_postgres.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh -l
+#
+source "$(dirname ${0})/common/common"
+
+#%
+#% OpenShift Deploy Helper
+#%
+#%   Intended for use with a pull request-based pipeline.
+#%   Suffixes incl.: pr-###.
+#%
+#% Usage:
+#%
+#%    ${THIS_FILE} [SUFFIX] [apply]
+#%
+#% Examples:
+#%
+#%   Provide a PR number. Defaults to a dry-run.
+#%   ${THIS_FILE} pr-0
+#%
+#%   Apply when satisfied.
+#%   ${THIS_FILE} pr-0 apply
+#%
+
+
+# Target project override for Dev or Prod deployments
+#
+PROJ_TARGET="${PROJ_TARGET:-${PROJ_DEV}}"
+
+# Prepare variables for backups
+NAME="backup-postgres-${NAME_APP}-${SUFFIX}"
+VERIFICATION_VOLUME_NAME="backup-verification-postgres-${NAME_APP}-${SUFFIX}"
+CLUSTER_NAME="patroni-${NAME_APP}-${SUFFIX}"
+IMAGE_NAMESPACE=${PROJ_TOOLS}
+EPHEMERAL_STORAGE=${EPHEMERAL_STORAGE:-'False'}
+# -# /-/_ : replace dash with underscore
+# -# ^^ to uppercase
+# -WPS_ENVIRONMENT_USERNAME_KEY="PATRONI_${NAME_APP}_${SUFFIX/-/_}_LEADER_USER"
+# -WPS_ENVIRONMENT_USERNAME_KEY=${WPS_ENVIRONMENT_USERNAME_KEY^^}
+# -WPS_ENVIRONMENT_PASSWORD_KEY="PATRONI_${NAME_APP}_${SUFFIX/-/_}_LEADER_PASSWORD"
+# -WPS_ENVIRONMENT_PASSWORD_KEY=${WPS_ENVIRONMENT_PASSWORD_KEY^^}
+WPS_ENVIRONMENT_USERNAME_KEY="PATRONI_${NAME_APP}_${SUFFIX}_LEADER_USER"
+# replace - with _
```

Comment:
I'm a bit confused about what's going on here. Why are those 4 lines commented out? And why do the dashes need to be replaced with underscores?

Reply:
Sorry - somehow forgot to push the change where those 4 lines are gone. (the 4 lines work in bash, but ubuntu defaults to dash).

The backup project looks for environment variables <SERVER>_USER and <SERVER>_PASSWORD  ; with all underscores. Just doesn't work otherwise.